### PR TITLE
Bigger nav icons, hamburger!

### PR DESCRIPTION
**Pull request by cecileboucheron**
🍔
@keybase/react-hackers

### CODE REVIEW

File: shared/menubar/index.desktop.js [hunk 1]
```diff
@@ -117,22 +116,21 @@ class MenubarRender extends React.Component<Props, State> {
...
     )
   }
 
   _renderDaemonHandshakeWait() {
-    const menuColor = this.state.showingMenu ? Styles.globalColors.black_50 : Styles.globalColors.black_50
     const text =
       this.props.daemonHandshakeState === 'waitingForWaiters'
-        ? `Connecting UI services to crypto engine... This may take a few seconds`
-        : `Starting up Keybase`
+        ? `Connecting interface to crypto engine... This may take a few seconds.`
```
Comment (songgao) [inline shared/menubar/index.desktop.js]: should this be "the interface"?

File: shared/settings/index.js [hunk 2]
```diff
@@ -57,7 +57,7 @@ const Connected = compose(
 // $FlowIssue lets fix this
 Connected.navigationOptions = {
   header: undefined,
-  title: 'Settings',
+  title: '',
```
Comment (songgao) [inline shared/settings/index.js]: Would you mind adding a comment here explaining this is for mobile and we intentionally have no title here on mobile?

File: shared/menubar/index.desktop.js (hunk 1 above)
Comment (cecileboucheron) [reply shared/menubar/index.desktop.js]: I think UI language can be article-free in certain cases: "Loading dependencies...", "Syncing folders...", etc.

File: shared/settings/index.js (hunk 2 above)
Comment (cecileboucheron) [reply shared/settings/index.js]: done!

**Pull request merged**